zulipterminal/ui_tools/boxes.py

```diff
@@ -76,7 +76,12 @@ def __init__(self, view: Any) -> None:
         super().__init__(self.main_view(True))
         self.model = view.model
         self.view = view
-
+        # global users_list  # Making it global so it can be accessed in the MessageBox class
+        # users_list = self.view.users
+        # global user_names_counter  # Making it global so it needs to be calculated only once
+        # user_names_counter = Counter(
+        #     user["full_name"] for user in users_list
+        # )  # Counting number of users having same name
```

Comment:
This commented code seems redundant. If you don't think it is necessary to the PR, then you may remove it.